packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -215,33 +202,57 @@ export class PhoenixObjects {
 
   /**
    * Process the Hits from the given parameters and get them as a geometry.
-   * @param hitsParams Parameters for the Hits.
+   * @param hitsParams Hit object. Must contain 'pos', the array of [x,y,z] positions, Can optionally contain extraInfo, which will be added to the resultant hit.
+   * @param type Tells Phoenix how to draw this - currently can be Point (default), or Line.
    * @returns Hits object.
    */
-  public static getHits(hitsParams: any): Object3D {
-    let positions: any[];
-    let hitsParamsClone: any;
-
-    // If the parameters is an object then take out 'pos' for hits positions
-    if (typeof hitsParams === 'object' && !Array.isArray(hitsParams)) {
-      positions = [hitsParams.pos];
-      hitsParamsClone = hitsParams;
-    } else {
-      positions = hitsParams;
-      hitsParamsClone = { pos: hitsParams };
+  public static getHits(hitsParams: [{ pos: []; type?: string }]): Object3D {
```

Comment:
This `hitsParams` is a single parameter and `type` is inside a single object in array. I am guessing that this doesn't parse `@returns` correctly because of an extra `@param` and throws the docs error.